src/Morphir/IR/Source.elm

```diff
@@ -73,6 +87,28 @@ type alias DataSourceName =
     Name
 
 
+{-| Represents a Literal type.
+The fields of a Literal are:
+
+  - BoolLiteral: A boolean literal
+  - StringLiteral: A string literal
+  - WholeNumberLiteral: A whole number literal
+  - FloatLiteral: A float literal
+  - DecimalLiteral: A decimal literal
+  - LocalDateLiteral: A local date literal
+  - LocalTimeLiteral: A local time literal
+
+-}
+type Literal
+    = BoolLiteral
+    | StringLiteral
+    | WholeNumberLiteral
+    | FloatLiteral
+    | DecimalLiteral
+    | LocalDateLiteral
+    | LocalTimeLiteral
```

Comment:
Why do we need to add literals here?

Reply:
They allow specifying literals as inputs and not just rowsets.